GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -167,6 +167,8 @@ public override string GVFSBinDirectoryName
             {
                 get { return "vfsforgit"; }
             }
+
+            public override int MaxPipePathLength => 104;

```

Comment:
Will you add a comment and point to some documentation for this limit?

Reply:
Sadly, the best source I have is this: https://unix.stackexchange.com/questions/367008/why-is-socket-path-length-limited-to-a-hundred-chars and of course the apple dev documentation link is dead.

Reply:
Okay thanks!  That is all I could find as well.

Reply:
`man unix` manpage has this in the very beginning:
```
DESCRIPTION
     The UNIX-domain protocol family is a collection of protocols that provides local (on-machine) interprocess communication through the normal socket(2) mechanisms.  The UNIX-domain family supports the SOCK_STREAM and SOCK_DGRAM socket types and uses filesystem pathnames for addressing.

ADDRESSING
     UNIX-domain addresses are variable-length filesystem pathnames of at most 104 characters.  The include file <sys/un.h> defines this address:

           struct sockaddr_un {
                   u_char  sun_len;
                   u_char  sun_family;
                   char    sun_path[104];
           };
```

Reply:
Found the link finally and added a comment for Mac platform.